appeals/web/src/server/appeals/question-page/question-page.mapper.js

```diff
@@ -0,0 +1,80 @@
+import { initialiseAndMapAppealData } from '#lib/mappers/appeal.mapper.js';
+import { initialiseAndMapLPAQData } from '#lib/mappers/lpaQuestionnaire.mapper.js';
+
+/**
+ *
+ * @param {string} appealId
+ * @param {string} lpaQId
+ * @returns
+ */
+
+export const lpaQBackLink = (appealId, lpaQId) => {
+	return {
+		text: 'Back to LPA questionnaire',
+		link: `/appeals-service/appeal-details/${appealId}/lpa-questionnaire/${lpaQId}`
+	};
+};
+
+/**
+ *
+ * @param {string} appealId
+ * @returns
+ */
+export const appealBackLink = (appealId) => {
+	return {
+		text: 'Back to LPA questionnaire',
+		link: `/appeals-service/appeal-details/${appealId}`
+	};
+};
+
+/**
+ *
+ * @param {string} question
+ * @param {{appeal: import("#appeals/appeal-details/appeal-details.service.js").Appeal}} data
+ * @param {string} currentRoute
+ * @param {import('../../app/auth/auth-session.service').SessionWithAuth} session
+ * @returns {Promise<import("#lib/mappers/appeal.mapper.js").InputInstruction[]>}
+ */
+export async function appealQuestionPage(question, data, currentRoute, session) {
+	const mappedData = await initialiseAndMapAppealData(data, currentRoute, session);
+
+	const instructions = mappedData.appeal;
+	return getInputInstructions(question, instructions);
+}
+
+/**
+ *
+ * @param {string} question
+ * @param {{lpaQ: import("#appeals/appeal-details/appeal-details.types.js").SingleLPAQuestionnaireResponse}} data
+ * @param {string} currentRoute
+ * @returns {Promise<import("#lib/mappers/appeal.mapper.js").InputInstruction[]>}
+ */
+export async function lpaQQuestionPage(question, data, currentRoute) {
```

Comment:
Will this page need the session to handle permissions? We could refactor and inject the individual permissions rather than the role, in a further iteration.

Reply:
Session is currently just used in Appeals Details to assign a user to the appeal.